automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/WorkspaceListPage.scala

```diff
@@ -28,6 +28,14 @@ class WorkspaceListPage(implicit webDriver: WebDriver) extends BaseFireCloudPage
   private def restrictedWorkspaceLabel(ns: String, n: String) = Label(s"restricted-$ns-$n")
 
   def clickCreateWorkspaceButton(expectDisabled: Boolean = false): Option[CreateWorkspaceModal] = {
+    /* wait for the create-workspace button to have loaded its billing project info. When expectDisabled=false,
+        this should be a noop, because the button text is "Create New Workspace" when enabled.
+        When expectDisabled=true, we need this wait. The button is disabled while loading billing info,
+        then remains disabled if billing info is loaded but has no valid projects. This method can't distinguish
+        between those two modes of disabled without looking at the text.
+     */
+    await text "Create New Workspace"
```

Comment:
The button text has "..." at the end, but I don't know if leaving it out will make a difference. Just thought I'd make a note.

Reply:
yup, the condition is really a "foo contains bar", not "foo equals bar".